pkg/registry/core/service/ipallocator/allocator.go

```diff
@@ -254,7 +254,7 @@ func calculateIPOffset(base *big.Int, ip net.IP) int {
 func RangeSize(subnet *net.IPNet) int64 {
 	ones, bits := subnet.Mask.Size()
 	if (bits - ones) >= 31 {
-		panic("masks greater than 31 bits are not supported")
+		panic("masks smaller than /2 for IPv4 or /98 for IPv6 are not supported.")
```

Comment:
That's a good point. Maybe shorter and longer is more appropriate for describing the "length" of the subnet mask. Let me know if you find these descriptions acceptable.